bbc/simorgh
# Enable Optimo features+analysis promos

Overall changes
======
- Allows rendering of Optimo promos in the Features+Analysis section of the page as these are now being returned from the OJ API on Test

Code changes
======
- Updates the FrostedGlassPromo to check for Optimo promos and apply the appropriate formatting for display
- Adds tests and snapshot updates to cater for the new Optimo checks

Testing
======
1. Visit http://localhost:7080/kyrgyz/articles/cy0pvvd9jvpo?renderer_env=test locally and ensure that the FrostedGlassPromo component renders in the right hand column
2. Visit http://localhost:7080/mundo/noticias-internacional-64681811?renderer_env=live and ensure that the FrostedGlassPromo continues to render for CPS assets

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
